src/Toolkit/Str.php

```diff
@@ -528,6 +528,21 @@ public static function from(string $string, string $needle, bool $caseInsensitiv
         return static::substr($string, $position);
     }
 
+    /**
+     * Add's _1 to a string or increment the ending number to allow _2, _3, etc
+     *
+     * @param string $string The string to increment
+     * @param int $first Start with
+     * @param string $separator Separator
+     * @return string
+     */
+    public static function increment(string $string, int $first = 1, string $separator = '_'): string
```

Comment:
I suggest we swap the arguments as the `$separator` may be changed more often than the starting number. Maybe we should also default to `-` as kebab case for identifiers is more commonly used in Kirby.

```suggestion
     * @param string $separator
     * @param int $first Starting number
     * @return string
     */
    public static function increment(string $string, string $separator = '-', int $first = 1): string
```